main.go

```diff
@@ -22,6 +22,7 @@ func main() {
 	ignoreCaseParameter := flag.Bool("f", false, "Fold lower case to upper case characters")
 	randomSortingParameter := flag.Bool("R", false, "shuffle, but group identical keys.")
 	reversePrintingParameter := flag.Bool("r", false, "Print result in reverse order")
+	quickSortParameter := flag.Bool("qsort", false, "Use quick sort, if the sort specifications allow")
```

Comment:
Which "sort specifications"?

Reply:
i dont know i just copied the original explanation

Reply:
Yes, you're more or less re-writing the original program, but the help is about *your new program*. I.e. if there aren't any "sort specifications" in *your new program*, you shall not write about them in the help text.